.github/workflows/codeql-analysis.yml

```diff
@@ -3,8 +3,14 @@ name: 'CodeQL'
 on:
   push:
     branches: [master]
+    paths-ignore:
+      - api/*
   pull_request:
     branches: [master]
+    paths-ignore:
+      - '**/*.md'
+    paths-ignore:
+      - 'api/**'
```

Comment:
u have the `paths-ignore` key there twice?

Reply:
```suggestion
    paths-ignore:
      - '**/*.md'
      - 'api/**'
```
I think you wanted to do this instead?

Reply:
Ah yeah. Good catch, my mistake 🤦‍♀️